Update architecture-center/docs/example-scenario/azure-virtual-desktop/azure-virtual-desktop-content.md (#4626)
MicrosoftDocs/architecture-center: docs/example-scenario/azure-virtual-desktop/azure-virtual-desktop-content.md

These considerations implement the pillars of the Azure Well-Architected Framework, which is a set of guiding tenets that can be used to improve the quality of a workload. For more information, see [Microsoft Azure Well-Architected Framework](/azure/well-architected/).

The Well-Architected Framework, for the Azure Virtual Desktop workload, offers an [assessment tool](/azure/well-architected/azure-virtual-desktop/assessment) to self-assess the readiness of your Azure Virtual Desktop workload in production; this is an effective tool to check your alignment to best practices defined in the [Microsoft Azure Well-Architected Framework](/azure/well-architected/).

### Reliability

Reliability ensures your application can meet the commitments you make to your customers. For more information, see [Overview of the reliability pillar](/azure/well-architected/reliability).

> - **Capacity Allocation**: To ensure guaranteed allocation of compute resources, you can request an [On-demand capacity reservation](/azure/virtual-machines/capacity-reservation-overview) with no term commitment and can be combined with reserved instances.

Reviewer: ```suggestion
- **Ensure capacity is reserved**: To ensure guaranteed allocation of compute resources, you can request an [On-demand capacity reservation](/azure/virtual-machines/capacity-reservation-overview) with no term commitment and can be combined with reserved instances.
```

> - **Intra-region Resiliency**: Use [Availability zones](/azure/reliability/availability-zones-overview) for Azure services that support them such as:

Reviewer: ```suggestion
- **Add Intra-region resiliency**: Use [Availability zones](/azure/reliability/availability-zones-overview) for Azure services that support them such as:
```

    - Virtual Machines (Session Hosts)
    - Azure Storage (FSLogix or App Attach). For more information, see [Azure Storage redundancy](/azure/storage/common/storage-redundancy).

> - **Business Continuity**: If [Availability zones](/azure/reliability/availability-zones-overview) do not meet your **RTO/RPO targets**, review the guidance on [Multiregion Business Continuity and Disaster Recovery (BCDR)](/azure/architecture/example-scenario/azure-virtual-desktop/azure-virtual-desktop-multi-region-bcdr) for Azure Virtual Desktop.

Reviewer: ```suggestion
- **Build a business continuity plan**: If [availability zones](/azure/reliability/availability-zones-overview) do not meet your RTO or RPO targets, review the guidance on [Multiregion Business Continuity and Disaster Recovery (BCDR)](/azure/architecture/example-scenario/azure-virtual-desktop/azure-virtual-desktop-multi-region-bcdr) for Azure Virtual Desktop.
```

### Security

Security provides assurances against deliberate attacks and the abuse of your valuable data and systems. For more information, see [Design review checklist for Security](/azure/well-architected/security/checklist).

Consider the following security-related factors when you deploy Azure Virtual Desktop.

> > [!NOTE]
> > For a more extensive list, see the guidance on [Security recommendations for Azure Virtual Desktop](/azure/virtual-desktop/security-recommendations)
> 

Reviewer: ```suggestion
```

> - **Antivirus Exclusions**: For profile solutions like FSLogix that mount virtual hard disk files, it's recommended to exclude those file extensions. For more information, see [Configure Antivirus file and folder exclusions](/fslogix/overview-prerequisites#configure-antivirus-file-and-folder-exclusions).

Reviewer: Are the exclusions for security or are they for reliability?  If that's a reliability (or performance) recommendation, please move it there.

Author: Moved, nice catch!

> - **Microsoft Entra**: Users can sign into Azure Virtual Desktop from anywhere using different devices and clients. To minimize the risk of unauthorized access and provide your organization with the ability to manage sign-in risks, [Enforce Microsoft Entra multifactor authentication using Conditional Access](/azure/virtual-desktop/set-up-mfa).

Reviewer: ```suggestion
- **Use Microsoft Entra ID**: Users can sign into Azure Virtual Desktop from anywhere using different devices and clients. To minimize the risk of unauthorized access and provide your organization with the ability to manage sign-in risks, [Enforce Microsoft Entra multifactor authentication using Conditional Access](/azure/virtual-desktop/set-up-mfa).
```

> > For a more extensive list, see the guidance on [Security recommendations for Azure Virtual Desktop](/azure/virtual-desktop/security-recommendations)

Reviewer: ```suggestion
> For more security recommendations, see the guidance on [Security recommendations for Azure Virtual Desktop](/azure/virtual-desktop/security-recommendations).
```

> - **Encryption**: By default, most Azure managed disks are encrypted at rest when persisting to the cloud. If your session hosts require more extensive encryption, like end-to-end encryption, review the guidance on [managed disk encryption options](/azure/virtual-machines/disk-encryption-overview) to protect stored data from unauthorized access.

Reviewer: ```suggestion
- **Use encryption**: By default, most Azure managed disks are encrypted at rest when persisting to the cloud. If your session hosts require more extensive encryption, like end-to-end encryption, review the guidance on [managed disk encryption options](/azure/virtual-machines/disk-encryption-overview) to protect stored data from unauthorized access.
```

> - **Private Networking**: If you require private connectivity to Azure Virtual Desktop resources, use [Azure Private Link with Azure Virtual Desktop](/azure/virtual-desktop/private-link-overview) to retain traffic between your virtual network and the service on the Microsoft Network rather than the public internet.

Reviewer: ```suggestion
- **Use private networking**: If you require private connectivity to Azure Virtual Desktop resources, use [Azure Private Link with Azure Virtual Desktop](/azure/virtual-desktop/private-link-overview) to constrain traffic between your virtual network and the service on the Microsoft Network.
```

> 

Reviewer: ```suggestion

> [!NOTE]
> For more security recommendations, see the guidance on [Security recommendations for Azure Virtual Desktop](/azure/virtual-desktop/security-recommendations).

```

### Cost Optimization

Cost optimization is about looking at ways to reduce unnecessary expenses and improve operational efficiencies. For more information, see [Overview of the cost optimization pillar](/azure/architecture/framework/cost/overview).

Consider the following cost-related factors when you deploy Azure Virtual Desktop.

> - **Multi-Session**: For workloads with identical compute requirements, generally pooled host pools, [Windows Enterprise multi-session](/azure/virtual-desktop/windows-multisession-faq) offers the ability to accept more users to sign in to a single VM at once; reducing costs and administrative overhead.

Reviewer: ```suggestion
- **Plan multi-session support**: For workloads with identical compute requirements, generally pooled host pools, [Windows Enterprise multi-session](/azure/virtual-desktop/windows-multisession-faq) offers the ability to accept more users to sign in to a single VM at once; reducing costs and administrative overhead.
```

> - **Licensing**: If you have Software Assurance, you can use [Azure Hybrid Benefit](/azure/virtual-machines/windows/hybrid-use-benefit-licensing) to reduce the cost of your Azure Compute Infrastructure.

Reviewer: ```suggestion
- **Optimize licensing**: If you have Software Assurance, you can use [Azure Hybrid Benefit](/azure/virtual-machines/windows/hybrid-use-benefit-licensing) to reduce the cost of your Azure compute infrastructure.
```

- **Reservations**: You can commit to one-year or three-year plans, [Azure Reservations](/azure/cost-management-billing/reservations/save-compute-costs-reservations), based on your VM usage to receive a discount to significantly reduce your resource cost. This can be combined with Azure Hybrid Benefit for up to 80 percent savings over list prices. 

> - **Scaling**: If committing to Azure Reservations is too demanding for your current needs, consider [Autoscale scaling plans](/azure/virtual-desktop/autoscale-scenarios) for power management or dynamic provisioning/deprovisioning of session hosts as the demand changes through the day/week.

Reviewer: ```suggestion
- **Scale in and out as needed**: If committing to Azure Reservations is not appropriate for your current needs, consider [Autoscale scaling plans](/azure/virtual-desktop/autoscale-scenarios) for dynamic provisioning/deprovisioning of session hosts as the demand changes through the day/week.
```

- **Load-Balancing Options**: If your organization is cost-conscious then configure your host pool load balancing algorithm to depth-first. However, this can degrade the users experience; the default breadth-first could be optimal if you value user experience. For more information, see [Configure host pool load balancing in Azure Virtual Desktop](/azure/virtual-desktop/configure-host-pool-load-balancing?tabs=portal)

### Operational Excellence

Operational Excellence covers the operations processes that deploy an application and keep it running in production. For more information, see [Design review checklist for Operational Excellence](/azure/well-architected/operational-excellence/checklist).

- **Alerts**:  Configure [Service Health](https://learn.microsoft.com/en-us/azure/service-health/service-health-portal-update) and [Resource Health](/azure/service-health/resource-health-overview) alerts to stay informed about the health of the Azure services and regions that you use.

>     - Monitor the Azure Storage solution that you use for hosting FSLogix Profiles or App Attach shares to ensure that thresholds aren't exceeded, which might have a negative impact on your user experience.

Reviewer: ```suggestion
  - Monitor the Azure Storage solution that you use for hosting FSLogix Profiles or App Attach shares to ensure that thresholds aren't exceeded, which might have a negative impact on your user experience.
```

- **Collect usage insights**: Use [Azure Virtual Desktop Insights](/azure/virtual-desktop/insights-use-cases) to help with checks such as which client versions are connecting, opportunities for cost saving, or knowing if you have resource limitations or connectivity issues.
- **Tune diagnostic settings**:  Enable diagnostic settings for all services, Azure Virtual Desktop workspaces, application groups, host pools, storage accounts. Determine which settings are meaningful to your operations. Turn off settings that aren't meaningful to avoid undue costs; storage accounts (specifically the file service) that see a high amount of IOPS can incur high monitoring costs.

### Performance Efficiency 

Performance efficiency is the ability of your workload to scale to meet the demands placed on it by users in an efficient manner. For more information, see [Performance Efficiency pillar overview](/azure/well-architected/performance-efficiency).

> - **Use Antivirus exclusions**: For profile solutions like FSLogix that mount virtual hard disk files, it's recommended to exclude those file extensions. For more information, see [Configure Antivirus file and folder exclusions](/fslogix/overview-prerequisites#configure-antivirus-file-and-folder-exclusions).

Reviewer: ```suggestion
- **Use antivirus exclusions**: For profile solutions like FSLogix that mount virtual hard disk files, it's recommended to exclude those file extensions. For more information, see [Configure Antivirus file and folder exclusions](/fslogix/overview-prerequisites#configure-antivirus-file-and-folder-exclusions).
```

> - **Latency**: For clients using a Point-to-Site (P2S) VPN connection use a split tunnel that's based on User Datagram Protocol (UDP) to reduce latency and optimize your tunnel bandwidth usage. For on-site clients who use a VPN or Azure ExpressRoute, use [RDP Shortpath](/azure/virtual-desktop/rdp-shortpath?tabs=public-networks) to reduce the round-trip time, which improves the user experience in latency-sensitive applications and input methods.

Reviewer: ```suggestion
- **Tune for latency**: For clients using a Point-to-Site (P2S) VPN connection use a split tunnel that's based on User Datagram Protocol (UDP) to reduce latency and optimize your tunnel bandwidth usage. For on-site clients who use a VPN or Azure ExpressRoute, use [RDP Shortpath](/azure/virtual-desktop/rdp-shortpath?tabs=public-networks) to reduce the round-trip time, which improves the user experience in latency-sensitive applications and input methods.
```

> - **Right-Sizing**: [Virtual machine sizing guidelines](/windows-server/remote/remote-desktop-services/virtual-machine-recs) lists the maximum suggested number of users per virtual central processing unit (vCPU) and minimum VM configurations for different workloads. This data helps estimate the VMs you need in your host pool.

Reviewer: ```suggestion
- **Use right-size compute**: [Virtual machine sizing guidelines](/windows-server/remote/remote-desktop-services/virtual-machine-recs) lists the maximum suggested number of users per virtual central processing unit (vCPU) and minimum VM configurations for different workloads. This data helps estimate the VMs you need in your host pool.
```

>     - Utilize simulation tools to test deployments with both stress tests and real-life usage simulations. Make sure that the system is responsive and resilient enough to meet user needs, and remember to vary the load sizes when testing.

Reviewer: ```suggestion
  - Utilize simulation tools to test deployments with both stress tests and real-life usage simulations. Make sure that the system is responsive and resilient enough to meet user needs and remember to vary the load sizes when testing.
```

For more information about Azure subscription limitations, see [Azure subscription and service limits, quotas, and constraints](/azure/azure-resource-manager/management/azure-subscription-service-limits).

## Deploy this scenario

A collection of [ARM templates](https://github.com/Azure/RDS-Templates/tree/master/ARM-wvd-templates) can be employed automate the deployment of your Azure Virtual Desktop environment. These ARM templates support only the Azure Resource Manager Azure Virtual Desktop objects. These ARM templates don't support Azure Virtual Desktop (classic).

More scenarios are available from Microsoft Developer Tools which supports several deployment options:

- [Azure Virtual Desktop with Microsoft Entra ID Join](https://learn.microsoft.com/en-us/samples/azure/azure-quickstart-templates/azure-virtual-desktop/)
- [Azure Virtual Desktop with FSLogix and AD DS Join](https://learn.microsoft.com/en-us/samples/azure/azure-quickstart-templates/azure-virtual-desktop-with-fslogix/)

> Consider [Azure Verified Modules](https://azure.github.io/Azure-Verified-Modules/) (AVM) to develop your own scenario using Bicep or Terraform.
> 

Reviewer: AAC doesn't typically recommend AVM without added context on why it would be more beneficial for this scenario to use AVM over pure native bicep or terraform.

```suggestion
```

Author: Understood, will bare this in mine for other AAC documents :) Thank you

## Next steps

- [Azure Virtual Desktop partner integrations](/azure/virtual-desktop/partners) lists approved Azure Virtual Desktop partner providers and independent software vendors.

> - Use the [Virtual Desktop Optimization Tool](https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool) to help optimize performance in a Windows 10/11 Enterprise VDI (virtual desktop infrastructure) environment.

Reviewer: ```suggestion
- Use the [Virtual Desktop Optimization Tool](https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool) to help optimize performance in a Windows 11 Enterprise VDI (virtual desktop infrastructure) environment.
```
